services/pexel_service.rb

```diff
@@ -0,0 +1,26 @@
+require 'faraday'
+require 'figaro'
+require 'dotenv'
+Dotenv.load
+
+class PexelService
+  def self.get_photo(pref)
+    json = self.get_json(pref)
+    { url: json[:photos].sample[:src][:large] }
```

Comment:
We had previously selected the :src//:original, maybe they don't always have a large option?